tests/fixtures/textbox.html

```diff
@@ -0,0 +1,5 @@
+<p>
```

Comment:
I'll see what I can do, but I don't know if I know how to do something like this anymore.